doc/material-migration.md

```diff
@@ -29,6 +29,17 @@ We have standardized the naming of styles and resource keys. This change aims to
 | MaterialOutlinedTextBoxBorderThicknessFocused | OutlinedTextBoxBorderThicknessFocused |
 | MaterialOutlinedTextBoxBorderThicknessPointerOver | OutlinedTextBoxBorderThicknessPointerOver |
 
+#### Removal of PasswordBox Border Padding Resources
+
+The following resource keys related to `PasswordBox` border padding have been removed:
+
+| Old Resource Key | New Resource Key | Value |
+| ---- | ---- |---- |
+| OutlinedPasswordBoxBorderPadding | ***REMOVED*** | 1 |
+| OutlinedPasswordBoxBorderPaddingPointerOver | ***REMOVED*** | 0 |
+| OutlinedPasswordBoxBorderPaddingFocused | ***REMOVED*** | 0 |
+| OutlinedPasswordBoxBorderThicknessPointerOver | OutlinedPasswordBoxBorderThicknessPointerOver | 1 |
```

Comment:
@Xiaoy312 I think the overall upgrading section should be reviewed/adjusted a bit more based on previous changes:
![image](https://github.com/user-attachments/assets/d665e322-bb49-46da-b3ac-0814438b782f)
What do you think ?

Reply:
It should be merged together.